pkg/client/client_v2_test.go

```diff
@@ -125,12 +129,19 @@ func TestClientV2_Checkin_Initial(t *testing.T) {
 	// receive the units
 	var unitsMu sync.Mutex
 	var units []*Unit
+	var gotFQDN bool
 	go func() {
 		for {
 			select {
 			case <-ctx.Done():
 				return
-			case change := <-validClient.UnitChanges():
+			case change := <-validClient.UnitChanged():
+				for _, t := range change.Triggers {
+					if t == TriggerFeature {
+						gotFQDN = change.Features.FQDN.Enabled
```

Comment:
Is it worth having a trigger per feature? 

It seems like as soon as we support more than one feature we will force every user of this package to have to iterate over all of them to find what changed. Having something like `TriggerFQDNFeature` instead seems like it would be more convenient, at the cost of an additional enum entry per feature.